docs/source/tutorials/06_using_your_own_models.rst

```diff
@@ -0,0 +1,578 @@
+
+.. _tutorial_using_your_trained_model_with_peekingduck:
+
+Using Your Trained Model with PeekingDuck
+=========================================
+
+This section will show you how to convert your trained model into a custom PeekingDuck node, and
+give an example of how you can integrate this node in a PeekingDuck pipeline. It assumes that you
+are already familiar with the process of creating custom nodes, covered in the earlier
+:ref:`custom node <tutorial_custom_nodes>` tutorial.
+
+Converting to a Custom Model Node
+---------------------------------
+
+First, let's create a new PeekingDuck project within the existing ``castings_project`` folder.
+
+.. admonition:: Terminal Session
+
+   | \ :blue:`[~user/castings_project]` \ > \ :green:`peekingduck init` \
+
+Next, we'll use the :greenbox:`peekingduck create-node` command to create a custom node:
+
+.. admonition:: Terminal Session
+
+   | \ :blue:`[~user/castings_project]` \ > \ :green:`peekingduck create-node` \ 
+   | Creating new custom node...
+   | Enter node directory relative to ~user/castings_project [src/custom_nodes]: \ :green:`⏎` \
+   | Select node type (input, augment, model, draw, dabble, output): \ :green:`model` \
+   | Enter node name [my_custom_node]: \ :green:`casting_classifier` \
+   | 
+   | Node directory:	~user/castings_project/src/custom_nodes
+   | Node type:	model
+   | Node name:	casting_classifier
+   | 
+   | Creating the following files:
+   |    Config file: ~user/castings_project/src/custom_nodes/configs/model/casting_classifier.yml
+   |    Script file: ~user/castings_project/src/custom_nodes/model/casting_classifier.py
+   | Proceed? [Y/n]: \ :green:`⏎` \
+   | Created node!
+
+The ``castings_project`` folder structure should now look like this:
+
+.. parsed-literal::
+
+   \ :blue:`castings_project/` \ |Blank|
+   ├── pipeline_config.yml
+   ├── train_classifier.py
+   ├── training_results.png
+   ├── \ :blue:`castings_data/` \ |Blank|
+   │     ├── \ :blue:`train/` \ |Blank|
+   │     ├── \ :blue:`validation/` \ |Blank|
+   │     └── \ :blue:`test/` \ |Blank|
+   ├── \ :blue:`src/` \ |Blank|
+   │     └── \ :blue:`custom_nodes/` \ |Blank|
+   │           ├── \ :blue:`configs/` \ |Blank|
+   │           │     └── \ :blue:`model/` \ |Blank|
+   │           │           └── casting_classifier.yml
+   │           └── \ :blue:`model/` \ |Blank|
+   │                 └── casting_classifier.py
+   └── \ :blue:`weights/` \ |Blank|
+         ├── keras_metadata.pb
+         ├── saved_model.pb
+         ├── \ :blue:`assets/` \ |Blank|
+         └── \ :blue:`variables/` \ |Blank|
+
+``castings_project`` now contains **two files** that we need to modify to implement our custom
+node.
+
+#. **src/custom_nodes/configs/model/casting_classifier.yml**:
+
+   ``casting_classifier.yml`` updated content:
+
+   .. code-block:: yaml
+      :linenos:
+
+      input: ["img"]
+      output: ["pred_label", "pred_score"]
+
+      weights_parent_dir: weights
```

Comment:
`weights` here is a relative path right? Can custom `weights_parent_dir` be a relative path instead of absolute?

Reply:
Yes it's a relative path, as this is meant to be a simple example for users.